src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  Routes,
+  Route,
+  HashRouter as Router,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/Loader/homePage/HomePage';
+import { PeoplePage } from './components/Loader/people/PeoplePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="people">
+          <Route index element={<PeoplePage />} />
+          <Route path=":personSlug?" element={<PeoplePage />} />
+        </Route>
+
+        <Route path="*" element={<h1 className="title">Page not found</h1>} />
```

Comment:
Not nessesary, but on task we should Implement NotFoundPage, as I understand, so just show the title probably is not enough